e2e/e2e_test.go

```diff
@@ -898,33 +922,42 @@ func Test_Hedera_Native_EVM_NFT_Transfer(t *testing.T) {
 	}
 
 	// Step 2 - Send the NFT Allowance for the Payer Account
-	_, err = sendNFTAllowance(setupEnv, nftID, setupEnv.Clients.Hedera.GetOperatorAccountID(), setupEnv.PayerAccount)
+	_, err = verify.SendNFTAllowance(setupEnv.Clients.Hedera, nftID, setupEnv.Clients.Hedera.GetOperatorAccountID(), setupEnv.PayerAccount)
 	if err != nil {
 		t.Fatalf("Failed to send Allowance for NFT [%s]. Error: [%s]", nftIDString, err)
 	}
 	signaturesStartTime := time.Now().UnixNano()
```

Comment:
I think that we can remove this part and use the `now` in `verify.TopicMessagesWithStartTime` as we are doing above.